### PR TITLE
New sorting option index

### PR DESCRIPTION
**Pull request by vicentemendozam**
Where
=====

* **Related Issue:** #35 

What
====
- New way to order the organizations

How
===
- New select with the Descending option in Organizations

Screenshots
===========
issue_35

Test
====
I added the test to order of organizations

Deployment
==========
- Remenber to do a `rake sunspot:reindex`

Warnings
========
- Nothing

### CODE REVIEW

File: app/controllers/admin/organizations_controller.rb [hunk 1]
```diff
@@ -25,8 +25,7 @@ def new
       @organization.user = User.new
     end
 
-    def edit
-    end
+    def edit() end
```
Comment (bertocq) [inline app/controllers/admin/organizations_controller.rb]: I think the `def edit; end` syntax is better here, was this a rubocop autocorrection or manual fix?

Comment (vicentemendozam) [reply app/controllers/admin/organizations_controller.rb]: I fixed manually to fix a rubocop offense but I'll change in the next commit.

Comment (bertocq) [reply app/controllers/admin/organizations_controller.rb]: You can just rebase the commit to fix it 👍

File: app/helpers/organizations_helper.rb [hunk 2]
```diff
@@ -2,4 +2,4 @@ module OrganizationsHelper
   def oganizations_index_subtitle
     t "organizations.subtitle.default" if params[:order].blank? || params[:order] == :created_at
   end
-end
+end
```
Comment (bertocq) [inline app/helpers/organizations_helper.rb]: Have you correctly configured http://editorconfig.org/ on your text editor or IDE? Please do so 🙏  https://codeclimate.com/github/AyuntamientoMadrid/agendas/app/helpers/organizations_helper.rb?from_sha=c8f2252c&to_sha=c2288a8b

**Comment by aitbw** [comment]
Updated PR accordingly, @bertocq. Sorry for the delay.

**Comment by aitbw** [comment]
FYI: There's room for improvement here, but should be included on a new PR.

The code for the `set_organization` and `search` methods found [here](https://github.com/AyuntamientoMadrid/agendas/blob/master/app/controllers/admin/organizations_controller.rb#L56) is also found [here](https://github.com/AyuntamientoMadrid/agendas/blob/master/app/controllers/organizations_controller.rb#L15), so we should extract it and make it a concern called `Searchable` (?), so we can DRY things up a little bit.

**Comment by taitus** [comment]
@aitbw 👍

**Pull request merged**